frontend/src/utils/victory.js

```diff
@@ -3,15 +3,26 @@ import { useCallback, useEffect, useMemo, useState } from 'react'
 export const getLineChartDomain = data => {
   const max = Math.max(...data.map(item => item.y))
   const min = Math.min(0, Math.min(...data.map(item => item.y)))
-  const margin = (max - min) * 0.05
-  return [min, max + margin]
+  return [min, max]
+}
+
+export const formatTickNumber = (value, formatNumber, axis) => {
+  if (axis.format === 'number') {
+    const { maximumFractionDigits, minimumIntegerDigits, minimumSignificantDigits, ...numberFormat } =
```

Comment:
Should be `minimumFractionDigits` instead of `maximumFractionDigits`. The goal is to remove any options which could make compactDisplay longer than expected.